Resources/ServerInfo/Guidebook/Antagonist/SpaceNinja.xml

```diff
@@ -58,19 +58,19 @@
 
   [bold]You have sworn yourself to the sword and refuse to use firearms.[/bold] Luckily, you've got a pretty cool sword.
 
-  Your [color=#66FF00]energy katana[/color] hurts plenty and can be [bold]recalled at will[/bold] at the cost of suit power. The farther away it is from you, the more energy required to recall it.
+  Your [color=cyan]energy katana[/color] hurts plenty and can be [bold]recalled at will[/bold] at the cost of suit power. The farther away it is from you, the more energy required to recall it.
 
-  While in hand you may also [color=#66FF00]teleport[/color] to anywhere that you can see, [bold]including through windows[/bold]. The charges required to do this regenerate slowly, so keep a charge or two spare in case you need a quick getaway.
+  While in hand you may also [color=cyan]teleport[/color] to anywhere that you can see, [bold]including through windows.[/bold] The charges required to do this regenerate slowly, so keep a charge or two spare in case you need a quick getaway.
```

Comment:
```suggestion
  While in hand you may also [color=cyan]teleport[/color] to anywhere that you can see, [bold]including through windows[/bold]. The charges required to do this regenerate slowly, so keep a charge or two spare in case you need a quick getaway.
```